sys/include/net/gnrc/nettype.h

```diff
@@ -129,6 +129,7 @@ typedef enum {
     /**
      * @}
      */
+    GNRC_NETTYPE_LORAWAN,       /**< Protocol is LoRaWAN */
```

Comment:
Please put this above `GNRC_NETTYPE_TEST`. Also it is necessary to surround this with a proper guard, otherwise memory is wasted, when `gnrc_lorawan` is not included in the firmware (the value space of `gnrc_nettype_t` decides the size of the `gnrc_netreg` registry at compile-time).

```suggestion
#ifdef MODULE_GNRC_LORAWAN
    GNRC_NETTYPE_LORAWAN,       /**< Protocol is LoRaWAN */
#endif
```

Reply:
addressed in https://github.com/RIOT-OS/RIOT/pull/11022/commits/610e8250a995b56c695b949ab4593af523188e3d